task/import.go

```diff
@@ -59,6 +61,12 @@ func TaskImport(tctx *TaskContext) (*data.TaskOutput, error) {
 		// TODO: Delete the uploaded file
 		return nil, err
 	}
+	// RecordStream (on our file) after async goroutines finish
```

Comment:
Is this accurate?
```suggestion
	// RecordStream (from our imported file) after async goroutines finish
```

Reply:
No, I will update the PR to do this however so that the `prepare/record` task works on the `imported` file on our end - similar to the pattern in `transcode`

Forgot to update this comment to the current state of the PR